src/components/sidebar/routePathView/routePathInfoTab/routePathInfoTab.scss

```diff
@@ -2,33 +2,33 @@
 @import '../../../shared/styles/flexbox';
 @import '../../../shared/styles/form';
 
-.routePathTab {
+.routePathInfoTabView {
     display: flex;
     flex-direction: column;
+    overflow-y: auto;
```

Comment:
We early took a decision to have all Save buttons sticky at the bottom. This change breaks that, and requires the user scrolling.

Reply:
Fixed, save button moved to form's parent. TODO's added.